### PR TITLE
Update the site to use sentence case consistently

### PR DESCRIPTION
**Pull request by bgravenorst**
Update the site to use sentence case consistently, also ensured that the topic headings matched the navigation.
Minor edits in a few places where admonitions where not closed off correctly.

Fixes: #432 and #436

### CODE REVIEW

File: docs/build-on-linea/tooling/cross-chain/axelar.md
Comment (jlwllmr) [inline docs/build-on-linea/tooling/cross-chain/axelar.md:1]: It seems as if the product here is just `Axelar` — perhaps we should amend the title to just that, and then correct usage of "Axelar Network" to something like "_the_ Axelar network"?

File: docs/build-on-linea/tooling/cross-chain/axelar.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
 ---
-title: Axelar Network
+title: Axelar network
 ---
 
-# Axelar Network
-
 The [Axelar Network](https://docs.axelar.dev/), is a blockchain of blockchains that allows for universal web3 interoperability. By integrating with Axelar, your Linea-based application now has access to the [45+ chains](https://axelarscan.io/) that are also connected via Axelar. As Axelar is a blockchain of blockchains it overcomes many of the challenges of more centralized interoperability solutions; transactions that go through the Axelar network cannot be censored by any oracle, relayer, or sequencer.
```
Comment (jlwllmr) [inline docs/build-on-linea/tooling/cross-chain/axelar.md:5]:
```suggestion
[Axelar](https://docs.axelar.dev/) is a blockchain of blockchains that allows for universal web3 interoperability. By integrating with Axelar, your Linea-based application now has access to the [45+ chains](https://axelarscan.io/) that are also connected via Axelar. As Axelar is a blockchain of blockchains it overcomes many of the challenges of more centralized interoperability solutions; transactions that go through the Axelar network cannot be censored by any oracle, relayer, or sequencer.
```

File: docs/build-on-linea/tooling/linea-safe.md
Comment (jlwllmr) [inline docs/build-on-linea/tooling/linea-safe.md:1]: was the intention to remove the `title` from the front matter as well here?

**Review by jlwllmr — APPROVED** [review]
Left a couple of comments — no changes necessary, but perhaps one or two recommended

File: docs/build-on-linea/tooling/cross-chain/axelar.md
Comment (bgravenorst) [reply docs/build-on-linea/tooling/cross-chain/axelar.md:1]: Yep, missed this. Thanks.

File: docs/build-on-linea/tooling/cross-chain/axelar.md (hunk 1 above)
Comment (bgravenorst) [reply docs/build-on-linea/tooling/cross-chain/axelar.md:5]: Agreed

File: docs/build-on-linea/tooling/linea-safe.md
Comment (bgravenorst) [reply docs/build-on-linea/tooling/linea-safe.md:1]: I may have been working on autopilot (with regards to how I add titles).
I'm fine user either the `title` meta or H1 heading to add titles, but I should probably have confirmed with you guys.

Comment (bgravenorst) [reply docs/build-on-linea/tooling/linea-safe.md:1]: I'll update it to use the meta tag instead

**Pull request merged**